pkg/engine/provider/filesystem_test.go

```diff
@@ -61,9 +61,11 @@ func TestFileSystemSourceProvider_GetSources(t *testing.T) { //nolint
 		excludes map[string][]os.FileInfo
 	}
 	type args struct {
-		ctx        context.Context
-		extensions model.Extensions
-		sink       Sink
+		ctx          context.Context
+		in1          string
```

Comment:
can you provide a better variable name in this case? couldn't understand to what it refers